packages/swapper/src/swappers/thorchain/types.ts

```diff
@@ -72,7 +72,10 @@ export interface BtcThorTrade<C extends ChainId> extends Trade<C> {
 }
 
 export interface EvmThorTrade<C extends ChainId> extends Trade<C> {
-  chainId: KnownChainIds.EthereumMainnet | KnownChainIds.AvalancheMainnet
+  chainId:
+    | KnownChainIds.EthereumMainnet
+    | KnownChainIds.AvalancheMainnet
+    | KnownChainIds.OptimismMainnet
```

Comment:
this isn't valid

Reply:
https://github.com/shapeshift/lib/pull/1178/commits/a195d63d67eae95cf35bd712bdfcc03c2aa3b662